gp-includes/template.php

```diff
@@ -253,7 +253,10 @@ function gp_project_links_from_root( $leaf_project ) {
 	$path_from_root = array_reverse( $leaf_project->path_to_root() );
 	$links[]        = empty( $path_from_root ) ? __( 'Projects', 'glotpress' ) : gp_link_get( gp_url( '/projects' ), __( 'Projects', 'glotpress' ) );
 	foreach ( $path_from_root as $project ) {
-		$links[] = gp_link_project_get( $project, esc_html( $project->name ) );
+		$link = gp_link_project_get( $project, esc_html( $project->name ) );
+		if ( '' !== strip_tags( $link ) ) {
+			$links[] = $link;
+		}
```

Comment:
I think instead of this, you could keep the existent logic, but only add the item to `$links[]` if the `project->id` isn't `null`.

```suggestion
		if ( ! is_null( $project->id ) ) {
			$links[] = gp_link_project_get( $project, esc_html( $project->name ) );
		}
```